docs/extend/modular-resources/create-subtype.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Define a New Resource Subtype"
+linkTitle: "New API Subtype"
+weight: 30
+type: "docs"
+tags: ["rdk", "extending viam", "modular resources", "API"]
+description: "Define an API for a new type of resource."
+no_list: true
+---
+
+You need to define a new resource *subtype* (and an API for that subtype) if:
+
+- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fit into any of the existing [component](/components/) or [service](/services/) subtypes.
+- You have a resource that could fit into an existing subtype, but you want to define an API with different endpoints than the ones in the standard Viam [API](/program/apis/) for that subtype.
+For example, you have a [sensor](/components/sensor/) and you want to define a `Calibrate` endpoint.
+
+{{% alert title="Tip" color="tip" %}}
+
+If you want to use most of an existing API but need just a few other functions, try using the [`DoCommand`](/program/apis/#docommand) endpoint and [extra parameters](/program/use-extra-params/) to add custom functionality to an existing subtype.
```

Comment:
It's just one option. People could technically do everything just with DoCommand and we don't even technically need different APIs at all, but that would be a pain. I think it's really up to the user which way they want to go. I think this tip is good as-is in that if you only want to add a little bit more to the API, this may be the way to go (but don't have to!)